pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/ClusterChangeHandler.java

```diff
@@ -18,10 +18,16 @@
  */
 package org.apache.pinot.broker.broker.helix;
 
+import org.apache.helix.HelixConstants;
+
+
 /**
  * Handles cluster changes such as external view changes, instance config changes, live instance changes etc.
  */
 public interface ClusterChangeHandler {
 
-  void processClusterChange();
+  /**
```

Comment:
please annotate this interface as an evolving application usable interface.

Reply:
Will do